src/new/wallet.js

```diff
@@ -1454,11 +1464,21 @@ class HathorWallet extends EventEmitter {
    *    'skipSpent': if should not include spent utxos (default true)
    *  }
    *
-   * @return {Array} Array of objects with {tx_id, index, address} of the authority output. Returns null in case there are no utxos for this type
+   * @return {Array} Array of objects with {txId, index, address} of the authority output. Returns an empty array in case there are no tx_outupts for this type
    **/
-  getMeltAuthority(tokenUid, options = {}) {
+  async getMeltAuthority(tokenUid, options = {}) {
     const newOptions = Object.assign({many: false, skipSpent: true}, options);
-    return this.selectAuthorityUtxo(tokenUid, wallet.isMeltOutput.bind(wallet), newOptions);
+    const txOutputs = this.selectAuthorityUtxo(tokenUid, wallet.isMeltOutput.bind(wallet), newOptions);
+
+    if (!txOutputs) {
```

Comment:
Maybe we can create a method with the lines from here and below. So if we decide to add a new key in the return we don't need to add in both methods

Reply:
We absolutely can, done in 3629080e5253bdc4439d33d9ef70ca66671335be